BookingsApi/BookingsApi.IntegrationTests/Api/V1/Hearings/BookNewHearingTests.cs

```diff
@@ -308,70 +305,12 @@ public async Task should_have_sent_relevant_judge_message_to_the_queue_when_a_he
         judgeMessage.Should().NotBeNull();
         _hearingIds.Add(response.Id);
     }
-
-    [Test]
-    public async Task should_not_send_judge_message_to_the_queue_when_a_new_participant_added_to_existing_hearing()
```

Comment:
Why is the scenario no longer being tested? If it's failing due to a non-existent flow shouldn't the assertions be updated